ci/jenkins/run-tests

```diff
@@ -13,8 +13,8 @@ pipeline {
             steps {
                 dir('liberty-tools-eclipse') {
                     // Clone git repo
-                    git branch: 'main',
-                        url: 'https://github.com/OpenLiberty/liberty-tools-eclipse.git'
+                    git branch: '1Q24-release',
+                        url: 'https://github.com/awisniew90/liberty-tools-eclipse.git'
```

Comment:
Looks like a test value

Reply:
right - just to test out our jenkins path.... in Jenkins, we can point to my forked branch to test with, but we clone from "main" for the test code. I put some thought into how we can change this, but it's not often enough we do this to warrant the time to fix.